src/components/DatePicker/DatePicker.js

```diff
@@ -0,0 +1,28 @@
+import GeneralStyles from '../../common/general.module.css';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function DatePicker({ editDate, startDate, onChange }) {
+  return (
+    <React.Fragment>
+      <input
+        disabled={editDate ? true : false}
+        type="date"
+        selected={startDate}
+        onChange={onChange}
+        value={editDate}
+        id="datepicker"
+      />
+      <label className={GeneralStyles.visuallyHidden} htmlFor="datepicker">
+        date picker
+      </label>
+    </React.Fragment>
+  );
+}
+DatePicker.propTypes = {
+  onChange: PropTypes.func,
+  editDate: PropTypes.string,
+  startDate: PropTypes.string
```

Comment:
I recommend you some better naming for these props.

This DatePicker to be truly reusable, should have agnostic prop names. So my suggestion:
- editDate -> disabled
- startDate -> value

That way you can reuse it other places of the app, even if not related with editDate/ startDate. Does this make sense to you?

Reply:
I will change it to disabled and value check if it will be confusing in the future